mLRS/CommonTx/esp.h

```diff
@@ -286,8 +289,10 @@ void tTxEspWifiBridge::EnterFlash(void)
 {
     if (!passthrough) return;
 
+#ifdef USE_DISPLAY
```

Comment:
good catch
isn't only for this internal PR, but should go into main independently, right?
if so might be nice to have an extra PR, as this PR here may need a bit longer

Reply:
Yes, it is more general than this pr, but doesn't seem worth a separate pr to me.  Since JLP is already developing on top of this code, hopefully I can get this cleaned up and you can merge it fairly soon.